Src/Common/Domain/service/null/NullOrganizationUserService.cs

```diff
@@ -19,24 +19,18 @@ public virtual void AddOrganizationUser(IOrganizationUser organizationUser)
         {
             // do nothing
         }
+
         public virtual IOrganizationUser GetOrganizationUser(int recordId)
         {
             return null;
-
-        }
-        public virtual IOrganizationUser GetOrganizationUser(int organizationId, int userId)
-        {
-            return null; 
-        }
-        public virtual int GetOrganizationUserRole(int organizationId, int userId)
-        {
-            return -1; 
         }
+
         public virtual IOrganizationUser[] GetOrganizationUsers()
```

Comment:
Wonder if we even want this. Don't change it now, but unless we want admins to see all of these types of users we probably won't need this method.